lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -12,17 +13,29 @@ module CLI
         # It depends on the syskit instance using log rotation
         class LogRuntimeArchive
             DEFAULT_MAX_ARCHIVE_SIZE = 10_000_000_000 # 10G
+            FREE_SPACE_LOW_LIMIT = 1_000_000_000 # 1 G
+            FREE_SPACE_FREED_LIMIT = 10_000_000_000 # 10 G
```

Comment:
The default archive size is 10G. I feel that these low/freed limits are on the low side.

The objective is that the free-space manager lets the archiver do its job. In practice, this means that within the default 10 min period, we need to generate less than freed_limit compressed data. We already do (we do ~10G/hour)

Let's baseline 20G/hour. That's 3G per 10 min. I think the low limit should be 5G and the freed limit 25G to let some room to breathe.